tests/integration-tests/src/test/kotlin/common/JwtCredentialProblem.kt

```diff
@@ -97,7 +96,7 @@ enum class JwtCredentialProblem {
             TODO("Not yet implemented")
         }
         override val verification = VcVerification.SUBJECT_VERIFICATION
-    };
+    }, ;
```

Comment:
This looks weird

Reply:
@FabioPinheiro I agree, but megalinter formatter has been adding this comma to the end. I don't like it either 😅